### ISSUE TITLE
`EventLoops.warmUp(EventLoopGroup)`

### ISSUE THREAD

**Issue by trustin** [line/armeria#3363]
Netty's event loop threads are spawned lazily, i.e. they are created on the first task is scheduled on. A user sometimes wants to ensure the event loop threads are started when the server starts up.

**Comment by kezhenxu94** [comment]
@trustin do you think it's a good idea to schedule a noop task to warm up the event loop group?

**Comment by trustin** [comment]
For now that's the only way to warm up an event loop group. Do you have a better idea?

**Comment by kezhenxu94** [comment]
> For now that's the only way to warm up an event loop group. Do you have a better idea?

That's also the only way I can come out with.

**Comment by trustin** [comment]
Yeah, we could just iterate over all event loops in an event loop group (using `iterator()`) and just submit a no-op task for each of them.
